main.py

```diff
@@ -37,6 +39,32 @@ def convert_website_to_stamp(_website, maximum_pages):
 
     LOGGER.debug(_website.convert_to_dict())
 
+    _classifier_and_summarizer = ClassifierAndSummarizer(
+        _website.contents,
+        maximum_pages
+    )
+
+    _classifier_and_summarizer_response \
+        = _classifier_and_summarizer.get_stampified_content()
+
+    if not _classifier_and_summarizer_response["is_stampifiable"]:
+        LOGGER.debug("Website is not stampifiable")
+        return
+
+    _generator = StampGenerator(
+            _website,
+            _classifier_and_summarizer_response["stamp_pages"]
+        )
+    generated_stamp = _generator.generate_stamp()
+
+    try:
+        website_title = _website.contents.content_list[0].text_string
```

Comment:
create get_title() in Website
And directly use _website.getTitle()

Reply:
@KomalBhadani27